.changeset/many-weeks-sort.md

```diff
@@ -0,0 +1,37 @@
+---
+'astro': minor
+---
+
+Form support in ViewTransitions router
+
+The `<ViewTransitions />` router can now handle form submissions, allowing you to retain stateful UI. This feature is opt-in for semver reasons and can be enabled by adding the `handleForms` prop to the component like so:
+
+```astro
+---
+// src/layouts/MainLayout.astro
+import { ViewTransitions } from 'astro:transitions';
+---
+
+<html>
+  <head>
+    <!-- ... -->
+    <ViewTransitions handleForms />
+  </head>
+  <body>
+    <!-- ... -->
+  </body>
+</html>
+```
+
+Like with links, you can opt-out of the router handling a submission on a per-form basis with the `data-astro-reload` property:
+
+```astro
+---
+// src/components/Contact.astro
+---
+<form class="contact-form" action="/request" method="post" data-astro-reload>
```

Comment:
```suggestion
<form class="contact-form" action="/request" method="POST" data-astro-reload>
```
Should these now be capitalized?

Reply:
I think it doesn't matter, but MDN documents them as lower case: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#method

Also vscode completions suggest lower case.

Reply:
OK, so we changed *our* case for v3 when writing JS in the frontmatter, but because this is just in HTML, we can use the lower case?  That's fine by me!